html-proofer.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |gem|
   gem.require_paths = ['lib']
 
   gem.add_dependency 'mercenary',       '~> 0.3.2'
-  gem.add_dependency 'nokogiri',        '~> 1.5'
+  gem.add_dependency 'nokogumbo',       '~> 1.4', '>= 1.4.10'
```

Comment:
This means *both* requirements apply. So it's `~> 1.4`, but at least `1.4.10`. That ensures that we can depend on the API from https://github.com/rubys/nokogumbo/pull/46.